lib/ruby_indexer/lib/ruby_indexer/configuration.rb

```diff
@@ -16,15 +16,19 @@ class Configuration
       T::Hash[String, T.untyped],
     )
 
+    sig { params(workspace_path: String).void }
+    attr_writer :workspace_path
+
     sig { void }
     def initialize
+      @workspace_path = T.let(Dir.pwd, String)
       @excluded_gems = T.let(initial_excluded_gems, T::Array[String])
       @included_gems = T.let([], T::Array[String])
       @excluded_patterns = T.let([File.join("**", "*_test.rb"), File.join("**", "tmp", "**", "*")], T::Array[String])
       path = Bundler.settings["path"]
-      @excluded_patterns << File.join(File.expand_path(path, Dir.pwd), "**", "*.rb") if path
+      @excluded_patterns << File.join(File.expand_path(path, @workspace_path), "**", "*.rb") if path
```

Comment:
Great catch. I switched the implementation to scope the glob patterns to the `@workspace_path` only when we get to computing `indexables`. Trying to initialize this object with the right path from the start is difficult because of Sorbet's strict requirement.